core/src/unprocessed_packet_batches.rs

```diff
@@ -51,6 +53,22 @@ pub struct ImmutableDeserializedPacket {
 }
 
 impl ImmutableDeserializedPacket {
+    pub fn new(
```

Comment:
nit: doesn't look like this `new` does much, not sure if necessary

Reply:
since we access outside the module we need it see `latest_unprocessed_votes`